### DISCUSSION TITLE
Theming Problem

### DISCUSSION THREAD

**Discussion by WebPirat** [unovue/shadcn-vue#599] [Q&A] [answered]
Good Morning,

i cant figure it out, i got the problem that all the components got differnt color css variables so the theming doesnt work :(
can someone give me advice what i did wrong ?

example bar-chart

```html
      <VisAxis
        v-if="showXAxis"
        type="x"
        :tick-format="xFormatter ?? ((v: number) => data[v]?.[index])"
        :grid-line="false"
        :tick-line="false"
        tick-text-color="hsl(var(--vis-text-color))"
      />
````
vis-text-color ?

**Comment by sadeghbarati** [comment, marked as answer]
https://www.shadcn-vue.com/docs/charts.html#update-css

Have you followed the Chart installation?

Can you share your reproduction?

```
@layer base {
  :root {
    /* ... */
    --vis-tooltip-background-color: none !important;
    --vis-tooltip-border-color: none !important;
    --vis-tooltip-text-color: none !important;
    --vis-tooltip-shadow-color: none !important;
    --vis-tooltip-backdrop-filter: none !important;
    --vis-tooltip-padding: none !important;

    --vis-primary-color: var(--primary);
    /* change to any hsl value you want */
    --vis-secondary-color: 160 81% 40%;
    --vis-text-color: var(--muted-foreground);
  }
 }
```

**Reply by WebPirat** [reply]
I am sorry it whas completly my mistake,..
your post is more or less the answer i had that css in the differnt file and the build tools whas a typo and he didnt load them

thanks for your time!
